### ISSUE TITLE
`<Icon />` accessibility improvement

### ISSUE THREAD

**Issue by travor20814** [Mezzanine-UI/mezzanine#167]
### Is your feature request related to a problem? Please describe.
To met web accessibility standards, `<svg />` should contains `<title>` which describe this image. 

### Describe the solution you'd like
Use icon.name to describe svg information (and should be customizable). And `aria-hidden` should be removed.

### Additional context or references
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title
